developer-docs-site/docs/guides/building-your-own-wallet.md

```diff
@@ -88,9 +88,12 @@ There will be some apis that certain wallets may add but there should be a few a
 ```typescript
 // Common Args and Responses
 
+// For single-signer account, there is one publicKey and minKeysRequired is null.
+// For multi-signer account, there are multiple publicKeys and minKeysRequired value.
 interface PublicAccount {
-    string address;
-    string publicKey;
+    address: string;
+    publicKeys: string[];
```

Comment:
since a majority of wallets are not multi-signer this feels like a large change to put onto every dapp. Also there could be multiple ways a multisigner wallet may be implemented. I think we should just do a `string | string[]` and when implementing something like verification.

Reply:
happy to have the discussion about it though

Reply:
Updated :)

Reply:
updated 👍